src/Raven.Embedded/ServerOptions.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Collections.Generic;
+
+namespace Raven.Embedded
+{
+    public class ServerOptions
+    {
+        public string FrameworkVersion { get; set; } = "2.1.0";
+
+        public string DataDir { get; set; }
+
+        public List<string> CommandLineArgs { get; set; } = new List<string>
```

Comment:
Please remember that in most of the time someone will write:
```
options.CommandLineArgs = new List<string> { "myArg1" };
```

and this will override your default (and mandatory) arguments.

Reply:
As I expected if we want to use the embedded in this way we will have to check every argument in the list 
or to make the user understand how to use it.
The other way is to remove commandLineArgs

Reply:
Please check what Oren did.